velox/functions/prestosql/benchmarks/ArrayMinMaxBasic.h

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "velox/expression/VectorFunction.h"
+#include "velox/functions/lib/LambdaFunctionUtil.h"
+#include "velox/vector/DecodedVector.h"
+#include "velox/vector/NullsBuilder.h"
+
+namespace facebook::velox::functions {
+namespace {
+
+template <typename T>
+class Min {
```

Comment:
isn't this just `std::less`?

Reply:
i copy pasted the vector function implementation here and then disabled the fast path

Reply:
will change it though

Reply:
i will change it though